src/AzureDevOpsDemoBuilder/Views/Extractor/Index.cshtml

```diff
@@ -317,9 +317,9 @@
 
         </div>
     </div>
-    <div class="footer">
-        We are moving to .Net Core 3.0, you are using preview version, log your issues <a class="text-white" href="https://github.com/microsoft/AzureDevOpsDemoGenerator/issues/new" target="_blank">here. We'll be return to main site after couple of hours</a>.
-    </div>
+    <!-- <div class="footer">
```

Comment:
Same as above...  But if this concept - an announcement header - is something you may reuse, consider making it a reusable component across pages which is empty unless there is an announcement to share.

Reply:
marked it as Reusable footer for announcement